pom.xml

```diff
@@ -20,1459 +20,1462 @@
 
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-    <modelVersion>4.0.0</modelVersion>
+  <modelVersion>4.0.0</modelVersion>
```

Comment:
Can you redo this file without all of the spacing changes and only your additions? It's a pain to see what has actually changed in this diff.